README.md

```diff
@@ -7,6 +7,7 @@ npm i OR yarn
 
 ## usage
 ```bash
```

Comment:
uh we want to change storage based on env variable not whole chunk of code in one place, look up some better patterns to encapsulate this please thanks